boards/bluepill/include/periph_conf.h

```diff
@@ -152,6 +152,41 @@ static const uart_conf_t uart_config[] = {
 #define UART_NUMOF          (sizeof(uart_config) / sizeof(uart_config[0]))
 /** @} */
 
+/**
+ * @name I2C configuration
+ * @{
+ */
+#define I2C_NUMOF           (2U)
+#define I2C_0_EN            1
+#define I2C_1_EN            0
```

Comment:
any reason why not enabling both I2C devices? At least I tested with both, works like charm 😉 Looking at the [pinout](http://wiki.stm32duino.com/images/a/ae/Bluepillpinout.gif) the given PIN config of both I2Cs is the default function of those pins anyway.

Reply:
We disabled the second I2C device because the Pins used by this device (PB10 and PB11) are also used by the USART3. We weren't sure how RIOT handles this and thus decided to disable the second I2C device by default since this is what seems to be done in most other STM32F1 based boards, e.g. the nucleon-f103.

Reply:
Yes, I know, but have you tried to use UART3, and is it working - because, as we discussed offline during HnA, I assume that UART3 is an alternative function of these PINs and must explicitly enabled, otherwise they are I2C2 anyway. I'll try to verify that too.

Reply:
mhm, UART3 is working as expected, too, so no need to activate an alternative function or something. Good then leave I2C_1 disabled but add a comment, that it collides with UART3.

Reply:
> Good then leave I2C_1 disabled but add a comment, that it collides with UART3.

Alright, will do that in a second.

Reply:
Pushed